test/cri-containerd/main.go

```diff
@@ -75,6 +75,7 @@ const (
 	featureLCOW           = "LCOW"
 	featureWCOWProcess    = "WCOWProcess"
 	featureWCOWHypervisor = "WCOWHypervisor"
+	featureHostProcess    = "HOSTPROCESS"
```

Comment:
Don't think this should be all caps?

Reply:
Yea, got thrown off by all (or portions) of the others being caps but they're all acronyms